packages/cli/src/commands/inspect.ts

```diff
@@ -212,3 +213,11 @@ function stateString(s: Deployment['readyState']) {
       return chalk.gray('UNKNOWN');
   }
 }
+
+function stripScheme(deploymentIdOrHost: string): string {
+  const match = deploymentIdOrHost.match(/http:\/\/|https:\/\//g)?.[0];
```

Comment:
This should probably use `^` to match that start because a url might have a query string too.

In fact, this should instead use `new URL(deploymentIdOrHost).hostname` in a try/catch instead to handle all these cases